src/convertToObject.js

```diff
@@ -3,16 +3,33 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const CONTAIN_LETTER_REGEX = /\w/;
+  const SPECIAL_SYMBOLS_REGEX = /\*+/g;
+  const styleObj = {};
+
+  const rules = sourceString
+    .split(';')
+    .filter(rule => rule.match(CONTAIN_LETTER_REGEX))
+    .map(rule => rule.replace(SPECIAL_SYMBOLS_REGEX, '').trim());
```

Comment:
Chaining is a good thing, but I recommend rewriting the code so that it is more readable. 
Also, try to use one array method which can return an object.